public/app/features/playlist/ShareModal.tsx

```diff
@@ -29,13 +29,17 @@ export const ShareModal = ({ playlistId, onDismiss }: ShareModalProps) => {
 
   const params: UrlQueryMap = {};
   if (mode) {
-    params.kiosk = mode;
+    params.kiosk = mode.toString();
```

Comment:
with all the redirects it somewhere loses the `?kiosk` param if it has no value. so instead of `https://local.logz.io/grafana-app/playlists/play/1?kiosk` it is now `https://local.logz.io/grafana-app/playlists/play/1?kiosk=true`